frontend/src/app/organization/organization-contact/organization-contact.component.scss

```diff
@@ -0,0 +1,9 @@
+label {
+  font-weight: 700;
+  text-align: right;
+}
+@media (max-width: 767.99px) {
```

Comment:
We decided (here: https://github.com/CodeForPoznan/volontulo/pull/807) that we will use bootstrap breakpoints.

Actually `767.98px` is one of them (see: https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints) so this should looks like:
```
@include media-breakpoint-down(sm) {
  label {
    text-align: left;
  }
}
```